samples/exposed-migration/README.md

```diff
@@ -0,0 +1,36 @@
+# Exposed Migration
+
+This sample project illustrates how to:
+
+- generate a migration script using Exposed
+- apply a migration using a third-party library like Flyway with a migration script generated by Exposed
+
+## Generating a migration script
+
+In `Tables.kt`, the table `UsersV1` represents the first version of the Users table, which does not have a primary key.
+The table `UsersV2` represents what `UsersV1` would look like if it is modified to have a primary key. You do not need
+to create new versions of your existing tables in your project. It is done this way in this sample project for
+demonstration purposes only. Differences between your modified Exposed tables and their corresponding definitions in the
+database will be detected by Exposed.
+
+The function `MigrationUtils.generateMigrationScript` generates a SQL migration script in the specified directory. The
+`exposed-migration` dependency is needed for this.
+
+```kotlin
+implementation("org.jetbrains.exposed:exposed-migration:$exposedVersion")
+```
+
+The generated migration script can also be edited manually before applying a migration.
+
+## Applying a migration
+
+In this sample project, Flyway is set up to apply migrations using migration scripts in the directory where Exposed
+generates them. It is recommended to apply the migration in a separate transaction.
```

Comment:
Maybe there could be a link to Flyway homepage here or to some official Flyway configuration reference doc?